dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -552,7 +606,17 @@ CFileDefExt::InitFileAttr(HWND hwndDlg)
             FileSize.u.LowPart = FileInfo.nFileSizeLow;
             FileSize.u.HighPart = FileInfo.nFileSizeHigh;
             if (SH_FormatFileSizeWithBytes(&FileSize, wszBuf, _countof(wszBuf)))
+            {
                 SetDlgItemTextW(hwndDlg, 14011, wszBuf);
+                
+                // Compute file on disk. If fails, use logical size
+                if (GetPhysicalFileSize(m_wszPath, &FileSize))
+                    SH_FormatFileSizeWithBytes(&FileSize, wszBuf, _countof(wszBuf));
+                else
+                    ERR("Unreliable size on disk");
```

Comment:
\n missing

Reply:
```suggestion
                    ERR("Unreliable size on disk\n");
```

Reply:
@learn-more But maybe instead of forcing developers for adding `\n`, we should just add `\n` in `ERR()` if string param does not include `\n` at the end of string?

Reply:
How stupid is this reply @Saibamen ... Why not adding `\n` automatically in all printf's if the string param does not include it, then?
Therefore breaking all code that does a two-step printf (here, ERR() ) if something needs to be shown appended to the string later on on purpose?

Reply:
New method `ERR_NEWLINE(string log)`

Reply:
This is longer than to just write `ERR("string log\n")` with the terminating `\n`  (just two characters instead of 8).

Reply:
OK, maybe I'm the only one using Visual Studio with IntelliSense...